src/main/scala/cromwell/Main.scala

```diff
@@ -42,7 +42,11 @@ object Main extends App {
    * Also now passing args to runAction instead of the constructor, as even sbt seemed to have issues with the args
    * array becoming null in "new Main(args)" when used with: sbt 'run run ...'
    */
-  val AskTimeout = Timeout(5 seconds)
+  // The magic maximum value for an ask timeout.  This value should be large to represent the amount of time until
+  // the future will fail (not the amount of time to wait for the Future to be returned from the ask).  This Future
+  // completes after the workflow goes to a terminal status, so the timeout must be at least as long as the workflow
+  // run duration.
+  val AskTimeout = Timeout(21474835 seconds)
```

Comment:
:+1: to @geoffjentry 

Right now timeouts are specified throughout `Main`, in:

- `Main$.<init>`
- `Main.run`
- `Main.runAction`
- `Main.runWorkflow`

One could consolidate the use of timeouts into a single line within `runWorkflow`:

```scala
val runner = workflowManagerSystem.actorSystem.actorOf(runnerProps, "SingleWorkflowRunnerActor")

val futureResult = runner.ask(RunWorkflow)(Timeout(21474835 seconds))
Await.ready(futureResult, Duration.Inf)
```

Alternatively, as was discussed, one could use a small custom `Actor`, just like the `PromiseActorRef`, but that doesn't timeout.

```scala
val runner = workflowManagerSystem.actorSystem.actorOf(runnerProps, "SingleWorkflowRunnerActor")

val promise = Promise[Any]()
workflowManagerSystem.actorSystem.actorOf(Props(classOf[PromiseWorkflowActor], promise, runner))
val futureResult = promise.future
Await.ready(futureResult, Duration.Inf)
```

Where the actor looks something like:

```scala
class PromiseWorkflowActor(promise: Promise[Any], runner: ActorRef) extends Actor {
  runner ! RunWorkflow

  override def receive = {
    case Status.Failure(f) => promise.tryFailure(f)
    case success => promise.trySuccess(success)
  }
}
```